src/rpc/mining.cpp

```diff
@@ -424,6 +424,7 @@ static RPCHelpMan getmininginfo()
                         {RPCResult::Type::NUM, "networkhashps", "The network hashes per second"},
                         {RPCResult::Type::NUM, "pooledtx", "The size of the mempool"},
                         {RPCResult::Type::STR, "chain", "current network name (" LIST_CHAIN_NAMES ")"},
+                        {RPCResult::Type::STR_HEX, "signet_challenge", /*optional=*/true, "The block challenge (aka. block script), in hexadecimal (only present if the current network is a signet)"},
```

Comment:
See above note about `-signetchallenge`.

Reply:
`signet_challenge` is consistent with `getblocktemplate`
https://github.com/bitcoin/bitcoin/blob/master/src/rpc/mining.cpp#L660